contrib/systemd/origin-node.sysconfig

```diff
@@ -13,10 +13,3 @@ CONFIG_FILE=/etc/origin/node/node-config.yaml
 # TODO: More elegant solution like this
 # https://github.com/coreos/flannel/blob/master/dist/mk-docker-opts.sh
 # DOCKER_NETWORK_OPTIONS='-b=lbr0 --mtu=1450'
-
-# Proxy configuration
```

Comment:
It would affect node -> docker that will always be local to the node will it not?

And as far as I know the kubelet doesn't propagate these into containers. I'll verify, we should probably come up with a good way to do that, I'm not sure that we're doing that outside of builddefaults/buildoverrides.